from benjaminhamon_blog.content.exceptions.content_exception import ContentException


class ContentNotFoundException(ContentException):
    pass
